flow/layers/layer_tree.cc

```diff
@@ -119,6 +119,16 @@ void LayerTree::Paint(CompositorContext::ScopedFrame& frame,
       internal_nodes_canvas.addCanvas(overlay_canvases[i]);
     }
   }
+  DisplayListBuilder* builder = frame.display_list_builder();
+  DisplayListBuilderMultiplexer builder_multiplexer;
+  if (builder) {
+    builder_multiplexer.addBuilder(builder);
+    if (frame.view_embedder()) {
+      for (auto view_builder : frame.view_embedder()->GetCurrentBuilders()) {
```

Comment:
Oh, sorry, miss here. If we use `auto`, it is better to specify whether it is a reference or a pointer.
Nit: change `auto` to `auto*`